eng/test-rebuild.ps1

```diff
@@ -36,40 +36,57 @@ try {
 
   $dotnetInstallDir = (InitializeDotNetCli -install:$true)
   $rebuildArgs = ("--verbose" +
-  " --assembliesPath `"$ArtifactsDir/obj/AnalyzerRunner/$configuration/netcoreapp3.1`"" +
```

Comment:
BTW, I ordered these lines alphabetically using a VS Code extension. So that should make the diffs more manageable when more are added in the future.